### PR TITLE
Prevent UploadGateway from being created when cloudServices#tokenUrl is not provided

### LINKED ISSUE #9: UploadGateway should not be created when `cloudServices#token` in not provided
I changed the behavior in https://github.com/ckeditor/ckeditor5-easy-image/pull/8 and move the check to the new cloud services package. But it turned out, that the CloudServices shouldn't throw an error when the `tokenUrl` config is not provided. So, in this case, all instances depending on the token shouldn't be created.

### LINKED ISSUE #2559: UploadGateway should not be created when `cloudServices#token` in not provided
I changed the behavior in https://github.com/ckeditor/ckeditor5-easy-image/pull/8 and move the check to the new cloud services package. But it turned out, that the CloudServices shouldn't throw an error when the `tokenUrl` config is not provided. So, in this case, all instances depending on the token shouldn't be created.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Prevented UploadGateway from being created when `cloudServices#tokenUrl` is not provided. Closes ckeditor/ckeditor5#2559.

### CODE REVIEW

File: tests/cloudservicesuploadadapter.js [hunk 1]
```diff
@@ -60,6 +60,20 @@ describe( 'CloudServicesUploadAdapter', () => {
...
 
+		it( 'should not set loader if there is no token', () => {
+			UploadGatewayMock.lastToken = undefined;
+
+			return ClassicTestEditor
+				.create( div, {
+					plugins: [ CloudServicesUploadAdapter ]
+				} )
+				.then( editor => {
+					expect( UploadGatewayMock.lastToken ).to.be.an( 'undefined' );
```
Comment (pomek) [inline tests/cloudservicesuploadadapter.js]: Why not `expect( UploadGatewayMock.lastToken ).to.equal( undefined )`?

Comment (ma2ciek) [reply tests/cloudservicesuploadadapter.js]: Hmm, that's the same as only `undefined` can give `"undefined"` is a `typeof something`. But I have to admit, that I just reverted this test from the past :D

Comment (szymonkups) [reply tests/cloudservicesuploadadapter.js]: Why not `.to.be.undefined`? :D

Comment (ma2ciek) [reply tests/cloudservicesuploadadapter.js]: Ok, I simplified it :D

**Pull request merged**